src/lib/svg_text_utils.js

```diff
@@ -400,7 +400,12 @@ function buildSVGText(containerNode, str) {
     }
 
     function exitNode(type) {
+        // A bare closing tag can't close the root node. If we encounter this it
+        // means there's an extra closing tag that can just be ignored:
+        if(nodeStack.length === 1) return;
```

Comment:
do you want to log this like we do below for `if(type !== innerNode.type)`? Though, we don't do anything with this so maybe we should get rid of them both...